schema/notifications.hcl

```diff
@@ -0,0 +1,34 @@
+table "notifications" {
+  schema = schema.public
+  column "id" {
+    null    = false
+    type    = uuid
+    default = sql("generate_ulid()")
+  }
+  column "events" {
+    null = false
+    type = sql("text[]")
+  }
+  column "template" {
+    null = false
+    type = text
+  }
+  column "receivers" {
+    null    = false
+    type    = jsonb
+  }
+  column "created_at" {
```

Comment:
need:

* created_by

foreign keys for: as we will need to show notification for a team/person

* team_id
* person_id 

* filter